src/Core/GoAspectContainer.php

```diff
@@ -91,12 +91,11 @@ public function __construct()
         $this->share('aspect.annotation.reader', function(Container $container) {
             $options = $container->get('kernel.options');
             $reader  = new AnnotationReader();
-            if (!empty($options['cacheDir'])) {
-                $reader = new FileCacheReader(
+            if (!empty($options['annotationCache'])) {
```

Comment:
I didn't fixed this one. 

`annotationCache` is not optional, however, current unit tests do not mocks engine configuration, so if I presume that `annotationCache` will always exist here, test fails.

In general - if you want, I can deal with test as well and remove if (!empty(...)) here, however, both code and tests should be revised -> what is required and what is optional...